src/ExtendedPdo.php

```diff
@@ -85,13 +85,26 @@ public function __construct(
         $this->setQuoteName($parts[0]);
     }
 
+    public static function connect(
+        string $dsn,
+        ?string $username = null,
+        ?string $password = null,
+        ?array $options = [],
+        array $queries = [],
+        ?ProfilerInterface $profiler = null
+    ): static {
+        $pdo = new static($dsn, $username, $password, $options ?? [], $queries, $profiler);
+        $pdo->establishConnection();
```

Comment:
Do we really want to connect immediately when using driver specific constructor? I'd say we always connect lazily.